src/main/webapp/modal-tester.html

```diff
@@ -14,8 +14,21 @@
         <button data-modal-close-button class="exit-button"> &times;</button>
       </div>
       <div class="modal-body">
-        Lorem ipsum dolor sit amet, Nessun dorma! Nessun dorma! Tu pure, oh Principessa. 
-        Nella tua fredda stanza Guardi le stelle che tremano. D'amore e di speranza.
+        <div id="results-list">
+          <ul>
+            <li>
+              <div class="result-grid"> 
+                <div class = "prvw-img"> </div>
+                <ul>
+                  <li> Phone Number! </li>
+                  <li> Address </li>
+                  <li> Hours of operation </li>
+                  <li> Rating </li>
+                  <li> Distance away </li>
```

Comment:
Not sure that this is a field we have access to, worth paying attention to not forget this.

Reply:
Modal Tester will be deleted in a future commit, as it is not needed to be client-facing, and will be better suited in documentation.